api/types.go

```diff
@@ -340,6 +340,9 @@ type App struct {
 	ImageDetails                ImageVersion
 	LatestImageDetails          ImageVersion
 
+	// TODO: seems wrong
+	Machines struct{ Nodes []GqlMachine }
```

Comment:
Less plugged into this work recently - should I just make a type scoped to this module?